tests/conftest.py

```diff
@@ -41,11 +143,24 @@ def app_logger(app):
 
 
 @pytest.fixture(scope='session')
-def initialized_db(app):
+def initialized_db(app, request):
     """Create database schema"""
+    db.drop_all()
     db.create_all()
 
```

Comment:
@waynelai1228 we shouldn't need a separate `teardown_db` fixture, especially as it's error prone to rely on all users of `initialized_db` to include the pair of fixtures.
If a fixture `yield`s , after the test is done, it'll run the code below the yield.  You should therefore be able to make the body of `teardown_db` part of `initialized_db`, just below a yield statement.